src/api/Gemfile.lock

```diff
@@ -1,3 +1,14 @@
+GIT
```

Comment:
We still can't do that. Are you going to resolve this or should we just close the PR until there is a new release?